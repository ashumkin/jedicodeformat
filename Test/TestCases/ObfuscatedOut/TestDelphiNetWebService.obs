unit testdelphinetwebservice;interface uses system.collections,system.componentmodel
,system.data,system.diagnostics,system.web,system.web.services;type twebservice1=
class(System.Web.Services.WebService){$REGION 'Designer Managed Code'}strict private
components:icontainer;procedure initializecomponent;{$ENDREGION}strict protected 
procedure dispose(disposing:boolean);override;private public constructor create;[
webmethod(description='A Sample Web Method')]function helloworld:string;end;implementation
{$REGION 'Designer Managed Code'}procedure twebservice1.initializecomponent;begin
end;{$ENDREGION}constructor twebservice1.create;begin inherited;initializecomponent
;end;procedure twebservice1.dispose(disposing:boolean);begin if disposing and(components
<>nil)then components.dispose;inherited dispose(disposing);end;function twebservice1
.helloworld:string;begin result:='Hello, interweb';end;end.