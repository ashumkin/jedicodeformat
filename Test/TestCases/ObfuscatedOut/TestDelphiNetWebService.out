unit testdelphinetwebservice;

interface

uses system.collections, system.componentmodel, system.Data, system.diagnostics, system.web, system.web.services;

type
  twebservice1 = class(System.Web.Services.WebService){$REGION 'Designer Managed Code'}
  strict private
    Components: icontainer;
    procedure initializecomponent;{$ENDREGION}
  strict protected
    procedure dispose(disposing: boolean); override;
  private
  public
    constructor Create;
    [webmethod(description = 'A Sample Web Method')]
    function helloworld: string;
  end;

implementation

{$REGION 'Designer Managed Code'}
procedure twebservice1.initializecomponent;
begin
end;{$ENDREGION}

constructor twebservice1.Create;
begin
  inherited;
  initializecomponent;
end;

procedure twebservice1.dispose(disposing: boolean);
begin
  if disposing and (Components <> nil) then
    Components.dispose;
  inherited dispose(disposing);
end;

function twebservice1.helloworld: string;
begin
  Result := 'Hello, interweb';
end;

end.
